configs/env-example

```diff
@@ -6,6 +6,7 @@ BUILD_ID=0
 API_PORT=3000
 APP_PORT=8080
 SWAGGER_ENABLED=true
+APPLICATION_ARCHIVE_DAYS = 43
```

Comment:
Please remove the extra spaces between the `43` to look like the other ones in the same file.